com.elsevier.MendeleyDesktop.json

```diff
@@ -0,0 +1,82 @@
+{
+    "app-id": "com.elsevier.MendeleyDesktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "branch": "stable",
+    "command": "mendeleydesktop",
+    "separate-locales": false,
+    "tags": ["proprietary"],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--device=all",
```

Comment:
Couldn't tell much about it from their website but what does this need all device access for? (And you can remove the previous line if it has `all`)

Reply:
Looks to me like it's not going to be needed.

Reply:
Yeah - this needs to be removed if not specifically justified.